src/components/main-form.js

```diff
@@ -217,199 +173,275 @@ class MainForm extends React.Component {
-            </Info>
-            <InputContainer>
-              <InputWrapper>
-                <label>Mortgage Option 1</label>
+                <Accordion title='How much can I afford?'>
+                  Most financial advisers agree that people should spend no more
+                  than <b>28 percent</b> of their gross monthly income on
+                  housing expenses and no more than <b>36 percent</b> on total
+                  debt.{' '}
+                  <a href='https://www.bankrate.com/calculators/mortgages/new-house-calculator.aspx'>
+                    Bankrate
+                  </a>{' '}
+                  has a great tool to help you determine how much you can
+                  afford.
+                </Accordion>
+              </FormGroup>
+              <FormGroup>
+                <h6>Loan Term</h6>
+                <Row>
+                  <Col>
+                    <label>Mortgage Option 1</label>
+                    <InputGroup>
+                      <Input
+                        name='options.option1.term'
+                        placeholder='Loan Term'
+                        type='number'
+                        value={option1.term}
+                        onChange={this.updateInput}
+                      />
+                      <InputGroupAddon addonType='append'>
+                        <InputGroupText>years</InputGroupText>
+                      </InputGroupAddon>
+                    </InputGroup>
+                  </Col>
+                  <Col>
+                    <label>Mortgage Option 2</label>
+                    <InputGroup>
+                      <Input
+                        name='options.option2.term'
+                        placeholder='Loan Term'
+                        type='number'
+                        value={option2.term}
+                        onChange={this.updateInput}
+                      />
+                      <InputGroupAddon addonType='append'>
+                        <InputGroupText>years</InputGroupText>
+                      </InputGroupAddon>
+                    </InputGroup>
+                  </Col>
+                </Row>
+                <Accordion title='Pros and Cons'>
+                  <a href='https://thelendersnetwork.com/15-year-mortgage-vs-30-year-mortgage-rates/'>
+                    The Lender’s Network
+                  </a>{' '}
+                  has an excellent article which outlines the pros and cons of
+                  15-year versus 30-year mortgages.
+                </Accordion>
+                <Accordion
+                  title={`What happens when after you’re done paying off the ${_.min(
+                    [option1.term, option2.term]
+                  )}-year mortgage?`}
+                >
+                  {/* TODO: Convert to dynamic numbers */}
+                  This tool assumes that you’ll invest the difference between
+                  the payment of the 15-year and the 30-year mortgages. If you
```

Comment:
@hdavidzhu can you make all mentions of 15-year, 30-year dynamic? Completely forgot about this